src/main/java/mate/academy/spring/mapper/impl/response/ShoppingCartResponseDtoMapper.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.mapper.impl.response;
+
+import java.util.stream.Collectors;
+import mate.academy.spring.mapper.DtoResponseMapper;
+import mate.academy.spring.model.ShoppingCart;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.dto.response.ShoppingCartResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ShoppingCartResponseDtoMapper
+        implements DtoResponseMapper<ShoppingCartResponseDto, ShoppingCart> {
+    @Override
+    public ShoppingCartResponseDto toDto(ShoppingCart shoppingCart) {
+        ShoppingCartResponseDto dto = new ShoppingCartResponseDto();
+        dto.setId(shoppingCart.getId());
+        dto.setUserId(shoppingCart.getUser().getId());
```

Comment:
```suggestion
        dto.setId(shoppingCart.getId());
        dto.setUserId(shoppingCart.getId());
```
Трохи швидше відпрацює